tests/app/template/test_rest.py

```diff
@@ -444,3 +426,20 @@ def test_should_return_all_template_versions_for_service_and_template_id(notify_
                     assert x['content'] == original_content + '1'
                 else:
                     assert x['content'] == original_content + '2'
+
+
+def test_update_does_not_create_new_version_when_there_is_no_change(notify_api, sample_template):
+    with notify_api.test_request_context():
+        with notify_api.test_client() as client:
+            auth_header = create_authorization_header()
+            data = {
+                'template_type': sample_template.template_type,
+                'content': sample_template.content,
+            }
+            resp = client.post('/service/{}/template/{}'.format(sample_template.service_id, sample_template.id),
+                               data=json.dumps(data),
+                               headers=[('Content-Type', 'application/json'), auth_header])
+            assert resp.status_code == 200
+    from app.dao.templates_dao import dao_get_template_by_id
```

Comment:
Pep 8 says:
"Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants."